kmk/scanners/digitalio.py

```diff
@@ -11,11 +11,13 @@ def __init__(
         cols,
         rows,
         diode_orientation=DiodeOrientation.COLUMNS,
+        resistor_pull=digitalio.Pull.DOWN,
```

Comment:
```suggestion
        pull=digitalio.Pull.DOWN,
```
Keep naming scheme consistent with `digitalio`.